src/typings/che.d.ts

```diff
@@ -115,14 +115,6 @@ declare namespace che {
     attributes: { [propName: string]: string };
   }
 
-  export interface ProjectSource {
```

Comment:
Why you remove it, is it only unused?
Please remove all unused here, or none =)

Reply:
It was outdate. Let us do it step by step.